### DISCUSSION TITLE
Referencing External tokenized paragraphs with --8<-- "Snippets" extension

### DISCUSSION THREAD

**Discussion by MikeLemo1** [facelessuser/pymdown-extensions#2088] [Q&A] [answered]
Hello all,
I looked for a way to include particular text paragraphs from other md files using "anchor reference set mode" and "line selection set mode"
as well as "Tokenized paragraph ref set mode".
Now I see that the snippets extension can do the first 2 very well with anchor selection:

`# --8<-- [start:func]
def my_function(var):
    pass
# --8<-- [end:func]
`
`--8<-- "example.py:func"
`

and line selection:

`--8<-- "file.md:4:6"`

Now I know mkdocs does some kind of tokenization of all the paragraphs in the md files that can be referenced say you have a paragraph that is named:

"# How to make a doc"

You'd reference it like #how-to-make-a-doc

Is it possible to tell snippets to copy a paragraph from an external file by just referencing the particular paragraph token? 
For convenience reasons similar to this syntax 

`--8<-- "docs/index.md#how-to-install"`

**Comment by facelessuser** [comment, marked as answer]
No, there is no way to do so at this time.

**Reply by MikeLemo1** [reply]
No easy and clear way to do it at this time at this place**
